package.nls.json

```diff
@@ -413,7 +413,7 @@
     "DataScience.selectJupyterInterpreter": "Select an Interpreter to start Jupyter",
     "DataScience.switchingKernelProgress": "Switching kernel to '{0}'",
     "DataScience.sessionStartFailedWithKernel": "Failed to start a session for the Kernel '{0}'. \nView Jupyter [log](command:{1}) for further details.",
-    "DataScience.kernelTimeout": "Timed out waiting to get a heartbeat from kernel process. \n{0} \nView Jupyter [log](command:{1}) for further details.",
+    "DataScience.kernelTimeout": "Timed out waiting to get a heartbeat from kernel process. \nView Jupyter [log](command:{0}) for further details.",
```

Comment:
Took a peek in the history and looks like nothing was ever hooked up for this first param. Didn't see anything too useful here to add for customers so I just removed it.

Reply:
Can we not use the command as a constant instead of formatting it at runtime?